### DISCUSSION TITLE
Multi spa in one capacitor app

### DISCUSSION THREAD

**Discussion by PAA81** [ionic-team/capacitor#3117] [Q&A]
Hi,

Currently we have an angularjs web app that is used as a web app on desktop (with html5 mode enabled)
and which is used as an android app with cordova (html5 mode disabled).

We want to migrate to angular step by step, so build new features on new angular app (that will have the same menu and top bar of the angularjs app).

We are testing the possibility of use single-spa in desktop browser and cordova environment.

In case this solution does not work, an approach for the mobile environment could be to embed 2 separate web apps in a single capacitor application.
If it is possible how to set up links between the 2 applications.
What do you think of this approach? Do you have maybe another approach? Thanks.

**Comment by imhoffd** [comment]
You may be possible to use [single-spa](https://single-spa.js.org/) (I haven't used it). Aside from that or a similar approach, I don't think it's possible.

**Reply by PAA81** [reply]
Okay, thank you very much for your answer.
So embed several spas in one capacitor application is not possible ?  thank you.

**Reply by ghost** [reply]
I am right now trying to implement capacitor in a single-spa application

**Comment by berndlackinger** [comment]
Yes it works, i even mixed remote an local spa-apps

**Reply by bywo** [reply]
@berndlackinger I'm interested in using Capacitor with a remote spa-app. How did you get it to work? Thanks!

**Comment by berndlackinger** [comment]
@bywo I added a local app which is packaged within the App that first does some checks and then redirects to the server hosted app.
